massa-models/src/hash.rs

```diff
@@ -1,41 +1,48 @@
-// Copyright (c) 2021 MASSA LABS <info@massa.net>
-
+use massa_signature::{PublicKey, Signature};
+use serde::{Deserialize, Serialize};
 use std::collections::{HashMap, HashSet};
-use std::convert::TryInto;
 use std::hash::{BuildHasherDefault, Hasher};
 use std::marker::PhantomData;
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct PubkeySig {
```

Comment:
I would not have put PubkeySig in this file...

Reply:
In a `pubkey.rs` file in `massa-models`?

Reply:
yes,  or in crypto, or in composite, as you prefer. All seem better-fit than prehashed.rs

Reply:
Fixed :heavy_check_mark: